symbolizer-ffi/README.md

```diff
@@ -0,0 +1,4 @@
+# Symbolizer-ffi
+
+This forwards parts of the blazesym C APIs.
+Please ensure changes to this folder can be upstreamed to [blazesym](https://github.com/libbpf/blazesym).
```

Comment:
Is there a reason we are doing this here rather than upstreaming directly?

Reply:
So we should upstream all changes.

The capis are here: 
https://github.com/libbpf/blazesym/blob/main/capi/Cargo.toml

This is just taking the bits that we need. Progressively, we can remove these in favour of higher level APIs with better integration with our current C APIs. However I think this is a good first step to see how good the library is.

We can not compose the C APIs (due to the current Cargo definition of how cbindgen works).